Graphics/MusicWheelItem Song NormalPart/get_lamp.lua

```diff
@@ -0,0 +1,86 @@
+local args = {...}
+local graphic = args[1]
+local pn = args[2]
+
+LampColors = {
+    [0]={1,1,1,0},
+    color "#ffffff",
+    color "#34bfff",
+    color "#48ff1d",
+    color "#fff568",
+    color "#bfeaff"
+}
+
+
+local function getLamp(self, song)
+    local best_lamp = 0
+
+    if song then
+        -- Default to the machine profile if not found
+        if PROFILEMAN:IsPersistentProfile(pn) then 
+          profile = PROFILEMAN:GetProfile(pn)
+        else 
+          profile = PROFILEMAN:GetMachineProfile()
```

Comment:
I don't think we should use `MachineProfile`. I think this feature should be restricted to people with profiles only.